system/logging/appenders/ConsoleAppender.cfc

```diff
@@ -1,62 +1,48 @@
-Date        :	04/12/2009
-Description :
-	A simple ConsoleAppender
-	
-Properties:
-- none
------------------------------------------------------------------------>
-<cfcomponent extends="coldbox.system.logging.AbstractAppender" 
-			 output="false"
-			 hint="A simple Console Appender">
-	
-	<!--- Init --->
-	<cffunction name="init" access="public" returntype="ConsoleAppender" hint="Constructor" output="false" >
-		<!--- ************************************************************* --->
-		<cfargument name="name" 		required="true" hint="The unique name for this appender."/>
-		<cfargument name="properties" 	required="false" default="#structnew()#" hint="A map of configuration properties for the appender"/>
-		<cfargument name="layout" 		required="false" default="" hint="The layout class to use in this appender for custom message rendering."/>
-		<cfargument name="levelMin"  	required="false" default="0" hint="The default log level for this appender, by default it is 0. Optional. ex: LogBox.logLevels.WARN"/>
-		<cfargument name="levelMax"  	required="false" default="4" hint="The default log level for this appender, by default it is 5. Optional. ex: LogBox.logLevels.WARN"/>
-		<!--- ************************************************************* --->
-		<cfscript>
-			// Init supertype
-			super.init(argumentCollection=arguments);
-			
-			instance.out = createObject("java","java.lang.System").out;
-						
-			return this;
-		</cfscript>
-	</cffunction>	
-	
-	<!--- Log Message --->
-	<cffunction name="logMessage" access="public" output="false" returntype="void" hint="Write an entry into the appender.">
-		<!--- ************************************************************* --->
-		<cfargument name="logEvent" type="any" required="true" hint="The logging event"/>
-		<!--- ************************************************************* --->
-		<cfscript>
-			var loge = arguments.logEvent;
-			var entry = "";
-			
-			if( hasCustomLayout() ){
-				entry = getCustomLayout().format(loge);
-			}
-			else{
-				entry = "#severityToString(loge.getseverity())# #loge.getCategory()# #loge.getmessage()# ExtraInfo: #loge.getextraInfoAsString()#";
-			}
-			
-			// Log message
-			instance.out.println(entry);
-		</cfscript>			   
-	</cffunction>
-	
-<!------------------------------------------- PRIVATE ------------------------------------------>
-	
-	
-</cfcomponent>
\ No newline at end of file
+ /**
+	*  Constructor
+	*
+	*  @name.hint        The unique name for this appender.
+	*  @properties.hint  A map of configuration properties for the appender.
+	*  @layout.hint      The layout class to use in this appender for custom message rendering.
+	*  @levelMin.hint    The default log level for this appender, by default it is 0. Optional. ex: LogBox.logLevels.WARN
+	*  @levelMax.hint    The default log level for this appender, by default it is 4. Optional. ex: LogBox.logLevels.WARN
+	*/
+	public ConsoleAppender function init(required string name,
```

Comment:
Is this the desired format even if there is only a single argument, or is it acceptable to keep single arguments on the same line?